apps/ai-content-generator/src/components/dialog/common-generator/field-selector/field-list/EntryFieldList.tsx

```diff
@@ -0,0 +1,34 @@
+import { FormControl, Select } from '@contentful/f36-components';
+import { Field } from '@hooks/dialog/useSupportedFields';
+import { ChangeEvent } from 'react';
+
+interface Props {
+  title: string;
+  selectedField: string;
+  fields: Field[];
+  onChange: (event: ChangeEvent<HTMLSelectElement>) => void;
+}
+
+function EntryFieldList(props: Props) {
+  const { title, selectedField, fields, onChange } = props;
+
+  const fieldList = fields.map((field) => (
+    <Select.Option key={field.name} value={field.name}>
+      {field.name}
+    </Select.Option>
+  ));
+
+  return (
+    <FormControl>
+      <FormControl.Label>{title}</FormControl.Label>
+      <Select value={selectedField} onChange={onChange}>
+        <Select.Option value="" isDisabled>
+          Select field...
+        </Select.Option>
```

Comment:
it looks to me when playing around with the branch that when the 'Select field...' is displayed, a user can still try and generate text. I think this might be remedied by a flow in Emily's designs ([example](https://www.figma.com/file/TQsQcvECzI6R2Jy2Utu6xP/AI-Content-Generator-Revision-2?type=design&node-id=176-66812&mode=design&t=YyulQ5JWpigk4fFj-4))

Reply:
Created a ticket on this!